src/fabric/ModalScreenNativeComponent.ts

```diff
@@ -13,6 +13,10 @@ import type {
 // eslint-disable-next-line @typescript-eslint/ban-types
 type ScreenEvent = Readonly<{}>;
 
+type ScreenTargetEvent = Readonly<{
```

Comment:
What is the reasoning behind this change here?

Reply:
This is how our public API looks:
```
  onAppear?: (e: NativeSyntheticEvent<TargetedEvent>) => void;
```
Thus, the change. I needed to write in on my won cause TargetedEvent is `{target: number}`, and Codegen doesn't like `number` :D

Reply:
We decided not to change it because updating NativeProps will produce unused properties in C++ - unnecessarily wasting memory. My understanding is that the target value in an event is filled by the ReactNative thus mismatch between NativeProps and React props.

Reply:
@kkafar If I missed something would be good to put it here, as I linked the discussion in comment.